samples/net/sockets/echo_service/sample.yaml

```diff
@@ -0,0 +1,18 @@
+sample:
+  description: echo server using socket service API
+  name: socket_service_echo
+common:
+  harness: net
+  depends_on: netif
+  filter: CONFIG_FULL_LIBC_SUPPORTED and not CONFIG_NATIVE_LIBC
+  # eventfd does not work properly with native_sim so exclude it here
+  platform_exclude:
+    - native_sim
+    - native_sim_64
```

Comment:
Note to self. Forgot to fix this, the sample app works just fine with native_sim so I will fix this.